amazon-ec2-net-utils.spec

```
@@ -38,7 +38,8 @@ BuildRequires: systemd-units
 Requires: systemd-units
 %endif # systemd
 Requires: dhclient
-Provides: ec2-net-utils
+Provides: ec2-net-utils >= 1.4
+Obsoletes: ec2-net-utils                                                                                                                
```

Obsoletes without an appropriate bound will cause problems in the future.

```suggestion
Obsoletes: ec2-net-utils < 1.4
```

(or whatever version is appropriate here)

---

Since the two packages share an origin, and thus use the same version history, would it not make sense to obsolete `ec2-net-utils < %{version}-%{release}` if that's the version of ec2-net-utils that we provide?

---

No, because there isn't an ec2-net-utils version that is `>= 1.4`, so there's no need to obsolete it.

As a general rule obsoletes should be as minimally-scoped as possible to reduce the likelihood of problems in the future.

---

OK. Minor detail, though, there is an ec2-net-utils = 1.4 (AL2 currently ships with ec2-net-utils-1.4-2.amzn2.noarch). So at the very least, I think it needs to be `Obsoletes: ec2-net-utils <= 1.4-2.amzn2`, but there's no guarantee that ec2-net-utils won't see more revisions before this package replaces it. We should probably update this package to something like 1.5-1 and obsolete ec2-net-utils < 1.5.

---

@iliana I've incorporated the change described above. Please take a look.